core_backend/app/contents/models.py

```diff
@@ -155,13 +156,7 @@ async def _get_content_embeddings(
     Vectorizes the content
     """
     text_to_embed = content.content_title + "\n" + content.content_text
-    content_embedding = embedding(
-        model=LITELLM_MODEL_EMBEDDING,
-        input=text_to_embed,
-        api_base=LITELLM_ENDPOINT,
-        api_key=LITELLM_API_KEY,
-    ).data[0]["embedding"]
-    return content_embedding
+    return await embedding(text_to_embed)
 
 
 async def get_similar_content(
```

Comment:
We should get rid of this function since  `get_similar_content` and  `get_similar_content_async` both use `aembedding`